modules/oc-mirror-v2-about.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected-v2.adoc
+// * updating/updating_a_cluster/updating_disconnected_cluster/mirroring-image-repository.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="installation-oc-mirror-v2-about_{context}"]
+= About the oc-mirror plugin V2
+
+You can use the oc-mirror OpenShift CLI (`oc`) plugin to mirror all required {product-title} content and other images to your mirror registry by using a single tool. 
+
+It provides the following features:
+
+* Provides a centralized method to mirror {product-title} releases, Operators, and other images.
+* Uses a declarative image set configuration file to include only the {product-title} releases, Operators, and images that your cluster needs.
+* Performs incremental mirroring, which reduces the size of future image sets.
+* Optionally generates supporting artifacts for OpenShift Update Service (OSUS) usage.
+* Use the existing workflow use case/s (as in v1)
+- Mirror to Disk
+- Disk to Mirror
+- Mirror to Mirror
+* Improve overall user experience
+* Support multi tier complex enclaves 
+- Partial or fully air-gapped
+- Hub and spoke type air-gapped environments
+- Generate archives (tar.gz diff)
+-- By time (i.e from first mirror to latest)
+-- Incremental
+- History metadata
```

Comment:
This is something oc-mirror uses internally, from my perspective this is not something customer should have access to.

cc: @sherine-k and @lmzuccarelli

Reply:
deleted